### PR TITLE
feat(inline-loader): add experimental colors

### LINKED ISSUE #1299: Experimental Component - Inline Loading - Development
**Development**
A developer can discover and install the experimental component by following accessible documentation.

- [ ] Add to carbon-components
- [ ] Update website (add to the experimental section and update component status page)

### PR DESCRIPTION
**Pull request by aledavila**
Closes #1299 

Inline Loading svg update

#### Changelog

**New**

- add circle path to svg 

http://inline-loading-x-quick-gelada.mybluemix.net/component/inline-loading--default.html

### CODE REVIEW

**Review by aagonzales — CHANGES_REQUESTED** [review]
Text should be `text-02`/ #565656 but other than that look good 👍

**Comment by aledavila** [comment]
@aagonzales updated

**Review by dakahn — APPROVED** [review]

**Review by aagonzales — APPROVED** [review]

**Review by asudoh — COMMENTED** [review]
LGTM 👍 basically, just wanted to ask if the older markup still works in stable mode. Thanks @aledavila!

**Comment by aledavila** [comment]
@asudoh yes it still works

File: src/components/inline-loading/inline-loading.hbs [hunk 1]
```diff
@@ -3,7 +3,8 @@
   <div class="{{@root.prefix}}--inline-loading__animation">
     <div data-inline-loading-spinner class="{{@root.prefix}}--loading {{@root.prefix}}--loading--small">
       <svg class="{{@root.prefix}}--loading__svg" viewBox="-75 -75 150 150">
-        <circle cx="0" cy="0" r="37.5"></circle>
+        <circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="37.5" />
```
Comment (joshblack) [inline src/components/inline-loading/inline-loading.hbs:7]: Should we add a check here like:

```hbs
{{#if content.componentsX}}
  <!-- New markup -->
{{#else}}
  <!-- Classic markup -->
{{/if}}
```

**Comment by aledavila** [comment]
@joshblack updated both loaders

File: src/components/loading/loading.hbs [hunk 2]
```diff
@@ -2,8 +2,10 @@
 <div data-loading class="{{@root.prefix}}--loading{{#if small}} {{@root.prefix}}--loading--small{{/if}}">
   <svg class="{{@root.prefix}}--loading__svg" viewBox="-75 -75 150 150">
     <title>Loading</title>
+    {{#if componentsX}}
     {{#if small}}<circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="37.5" />{{/if}}
-    <circle class="{{@root.prefix}}--loading__stroke" cx="0" cy="0" r="37.5" />
+    {{/if}}
+    <circle {{#if componentsX}}class="{{@root.prefix}}--loading__stroke"{{/if}} cx="0" cy="0" r="37.5" />
```
Comment (asudoh) [inline src/components/loading/loading.hbs:8]: Wanted to double-check if we are making `bx--loading__stroke` experimental only.

File: src/components/loading/loading.hbs [hunk 3]
```diff
@@ -2,8 +2,10 @@
 <div data-loading class="{{@root.prefix}}--loading{{#if small}} {{@root.prefix}}--loading--small{{/if}}">
   <svg class="{{@root.prefix}}--loading__svg" viewBox="-75 -75 150 150">
     <title>Loading</title>
+    {{#if componentsX}}
```
Comment (asudoh) [inline src/components/loading/loading.hbs:5]: Wanted to double-check if we are making `<circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="37.5" />` experimental only.

**Review by asudoh — COMMENTED** [review]
Thanks you for your effort testing older markup @aledavila! Just a couple of comments to your latest code.

File: src/components/loading/loading.hbs (hunk 2 above)
Comment (aledavila) [reply src/components/loading/loading.hbs:8]: this was a class I added in during the experimental changes of the loading therefore would be a breaking change. So for now yes should be experimental only same with background as they are new classes not present in older markup

Comment (aledavila) [reply src/components/loading/loading.hbs:8]: It was my mistake in first round forgetting to add the if statement

**Comment by asudoh** [comment]
@aledavila Thank you for clarifying!

**Pull request merged**